app.js

```diff
@@ -1,25 +1,37 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const logger = require("morgan");
+const cors = require("cors");
+const dotenv = require("dotenv");
+const mongoose = require("mongoose");
 
-const contactsRouter = require('./routes/api/contacts')
+const { notFound, serverError } = require("./utils");
 
-const app = express()
+dotenv.config({ path: "./.env" });
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
+const contactsRouter = require("./routes/api/contacts");
 
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
+const app = express();
 
-app.use('/api/contacts', contactsRouter)
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 
-app.use((req, res) => {
-  res.status(404).json({ message: 'Not found' })
-})
+mongoose
+  .connect(process.env.MONGO_URL || "mongodb://127.0.0.1:27017/db-contacts")
```

Comment:
```suggestion
  .connect(process.env.MONGO_URL)
```
Досить сумнівна практика